content/docs/1.10.0/nodes-and-volumes/nodes/multidisk.md

```diff
@@ -5,47 +5,111 @@ weight: 4
+  1. Head to the `Node` tab, select one of the nodes, and select `Edit Disks` in the dropdown menu.
+  2. Add the path of the mounted disk into the disk list of the node.
 
-> **Note**:
-> Since Longhorn uses filesystem ID to detect duplicate mounts of the same filesystem, you cannot add a disk that has the same filesystem ID as an existing disk on the same node.
-> See more details at https://github.com/longhorn/longhorn/issues/2477
+- **Using `kubectl` Command**
+
+  1. Execute `kubectl edit node.longhorn.io <node-name>` to edit the node resource.
+  2. Add the disk path to `spec.disks` in the node resource. For example:
+
+    ```yaml
+    ...
+    spec:
+      ...
+      disks:
+        ...
+        example-disk:
+          allowScheduling: true
+          diskDriver: ""
+          diskType: filesystem
+          evictionRequested: false
+          path: /mnt/example-disk
+          storageReserved: 0
+          tags: []
+    ...
+    ```
+
+  3. Save and exit the editor.
+
+Once a disk is added:
+
+- Longhorn automatically detects the storage details of the disk, such as maximum and available capacity.
+- If the disk is suitable for storing volume data, Longhorn begins scheduling volumes to it.
+
+Optionally, you can use the `Space Reserved` field in the UI or `spec.disks.<disk-name>.storageReserved` to reserve a portion of disk space (in bytes) for other purposes. This reserved space will not be used by Longhorn for volume data.
+
+To maintain node stability when compute resources (for example memory or disk) are under pressure, the `kubelet` requires some space to remain free. If these critical resources are exhausted, it can lead to node instability.
+
+Longhorn **reserves 30% of the root disk space (`/var/lib/longhorn`) by default** to prevent issues like `DiskPressure` conditions from the `kubelet`, particularly after scheduling multiple volumes. This behaviour is controlled by the `storage-reserved-percentage-for-default-disk` setting.
+
+> **Notice**:
+>
+> 1. You cannot add a disk path that is already in use by another Longhorn disk.
+> 2. Longhorn uses the filesystem ID to detect duplicate mounts. Therefore, you cannot add a disk with the **same filesystem ID** as another disk on the same node.  
+>    See: [Issue #2477](https://github.com/longhorn/longhorn/issues/2477)
 
 ### Use an Alternative Path for a Disk on the Node
 
-If you don't want to use the original mount path of a disk on the node, you can use `mount --bind` to create an alternative/alias path for the disk, then use it with Longhorn. Notice that soft link `ln -s` won't work since it will not get populated correctly inside the pod.
+If you prefer not to use the original mount path of a disk on the node, you can create an alternative (alias) path using `mount --bind` and configure Longhorn to use it. Note that using a symbolic link (`ln -s`) will not work because it won't be correctly recognized inside the pod.
```

Comment:
```suggestion
If you prefer to use a different path for a disk (rather than the original mount point), you can use `mount --bind` to create an alternative path. Do **not** use symbolic link (`lh -s`), as these are not properly resolved inside Longhorn pods.
```